backend/src/agents/config/tools.yaml

```
@@ -0,0 +1,95 @@
+    You can pass either an ID (e.g., {"selector": "#my-id"}) or a class (e.g., {"selector": ".my-class"}).   
+
+
+select_dropdown_tool:
+  name: "Select Dropdown Option"
+  description: >-
+    Selects an option from a dropdown using a CSS selector.
+    you can select by value (example- {"selector": "#my-select", "option_value": "1"}), label (e.g., {"selector": "#my-select", "option_label": "Option 1"}),
+    or index (e.g., {"selector": "#my-select", "option_index": 2}).
+
+
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (3 > 2)

(empty-lines)